src/components/Editor/Tabs.js

```diff
@@ -418,30 +415,6 @@ class SourceTabs extends PureComponent {
     );
   }
 
-  renderNewButton() {
-    const newTabTooltip = L10N.getFormatStr(
-      "sourceTabs.newTabButtonTooltip",
```

Comment:
can we get rid of this property in `assets/panel/...properties`

Reply:
That's the one I wasn't sure about! Its removed now.
Thanks Jason!